### PR TITLE
Add PyCon UK 2015 videos

### LINKED ISSUE #34: PyCon UK 2015
PyCon UK 2015

|  |  |
| --- | --- |
| **Web** | http://2015.pyconuk.org/ |
| **Playlist** | Part of https://www.youtube.com/channel/UChA9XP_feY1-1oSy2L7acog/videos |
|  |  |

### PR DESCRIPTION
**Pull request by redapple**
Fixes #34 

Talks data scraped from http://2015.pyconuk.org/schedule/flat/

I omitted 2 videos:
- https://www.youtube.com/watch?v=3FmLlRdEnKw
- https://www.youtube.com/watch?v=869XPPdiju0

### CODE REVIEW

**Comment by redapple** [comment]
Sorry, https://github.com/pyvideo/data/pull/122/files#diff-f72779430bad5c97eebf1669cccbd4d1 looks wrong.
https://www.youtube.com/watch?v=lR8uTPcVTlk looks like it was http://2015.pyconuk.org/abstracts/#panel

File: pycon-uk-2015/videos/pycon-uk-2015-lightning-pykids.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "description": "In the most exciting part of the conference, we see what the PyKids have been doing in their own track, not to be missed.",
+  "duration": 1603,
+  "recorded": "2015-09-19",
+  "speakers": [],
```
Comment (logston) [inline pycon-uk-2015/videos/pycon-uk-2015-lightning-pykids.json]: Can you add "Various speakers"

File: pycon-uk-2015/videos/pycon-uk-2015-python-powered-nukes.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "description": "NukeStudio is a digital compositing tool used throughout the visual\neffects industry. Our talk will try to cover how our Python bindings let\nusers use NukeStudio as an environment for making their own tools for\nvideo processing and compositing. This will involve going through a\ncouple of examples before going onto discussing the structure/design of\nthe bindings and how they also enable us to use NukeStudio in other\ncontexts.",
+  "duration": 1647,
+  "recorded": "2015-09-20",
+  "speakers": [
+    "Babak Khataee and Dan Milburn"
```
Comment (logston) [inline pycon-uk-2015/videos/pycon-uk-2015-python-powered-nukes.json]: Can we break this up to two elements?

File: pycon-uk-2015/videos/pycon-uk-2015-saturday-lightening-talk-show.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+  "description": "Saturday Lightening Talk Show\n\nIntroduced by the Lightening Talk Man",
+  "duration": 4240,
+  "recorded": "2015-09-19",
+  "speakers": [],
```
Comment (logston) [inline pycon-uk-2015/videos/pycon-uk-2015-saturday-lightening-talk-show.json]: Can we add "various speakers" here?

File: pycon-uk-2015/videos/pycon-uk-2015-saturday-morning-plenary-session.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
+{
+  "description": "PyCon UK acting chairman Zeth Green welcomes the delegates to the second day of the conference.",
+  "duration": 503,
+  "recorded": "2015-09-19",
+  "speakers": [],
```
Comment (logston) [inline pycon-uk-2015/videos/pycon-uk-2015-saturday-morning-plenary-session.json]: Can we add "various speakers" here?

File: pycon-uk-2015/videos/pycon-uk-2015-sunday-lightening-talks-and-the-non-closing-closing.json [hunk 5]
```diff
@@ -0,0 +1,14 @@
+{
+  "description": "Sunday Lightening Talks\nIntroduced by The Lightening Talk Man.\n\nThe Non-Closing Closing\nHere we end the first-part of the conference, and move our focus into the sprints. It is the closing because the crew now let their hair down and become normal sprinters but it is non-closing because half of the delegates are still there the next day. There is no spoon!",
+  "duration": 6083,
+  "recorded": "2015-09-20",
+  "speakers": [],
```
Comment (logston) [inline pycon-uk-2015/videos/pycon-uk-2015-sunday-lightening-talks-and-the-non-closing-closing.json]: Can we add "various speakers" here?

File: pycon-uk-2015/videos/pycon-uk-2015-sunday-morning-plenary.json [hunk 6]
```diff
@@ -0,0 +1,14 @@
+{
+  "description": "Zeth warbling about nothing in particular to fill time.",
+  "duration": 406,
+  "recorded": "2015-09-20",
+  "speakers": [],
```
Comment (logston) [inline pycon-uk-2015/videos/pycon-uk-2015-sunday-morning-plenary.json]: Can we add "various speakers" here?

**Comment by redapple** [comment]
Thanks @logston . I updated speakers as requested.

**Comment by logston** [comment]
Sweet. Thanks!

**Pull request merged**